tools/moonbeam-xcmp.ts

```diff
@@ -84,6 +84,13 @@ async function test() {
     "wrong balance for relayAlice, expected: 1000000000000000000, returned: " +
       Number(relayAlice.data.free)
   );
+  // ensure both parachains are registered on the relay chain
```

Comment:
@joelamouche I need to make sure that the parachains are correctly registered. So I'm trying to check to see if `ParaLifecycles` storage of the `paras` pallet has `Some(ParaLifeCycle)` for the parachains `200, 201` and is `None` for parachain `199`.

Is there a function in TS to check if something `isSome()`?

Reply:
Because the way I'm doing it now isn't working. It seems the queried storage item is never equal to undefined. I assumed if it was `None` then it would be equal to `undefined` but that assumption must be incorrect because it is throwing the first error every time. 

So the others pass, but it doesn't mean they're `Some(ParaLifeCycle)`.

Reply:
I think I know how to do it right. Using toHuman() and null.

Reply:
Here is how to do type checking in TS: https://medium.com/ovrsea/checking-the-type-of-an-object-in-typescript-the-type-guards-24d98d9119b0
TLDR: just check the existence of a field that is specific to the type you're looking for... `if (ParaLifeCycle.specificfield){}`
Not as magic as in rust...